versioneer.py

```diff
@@ -1,469 +1,1143 @@
-#! /usr/bin/python
 
-"""versioneer.py
+# Version: 0.17
```

Comment:
This isn't the `versioneer` library itself, [python-versioneer](https://github.com/warner/python-versioneer) generates a `versioneer.py` to the root of the project repository, so that `setup.py` can use it to compute versions (I guess it was easiest to do this way in order to perform heuristics with the repository directory). In fact `versioneer.py` doesn't exist in the python-versioneer repo. I'll add some additional context to this PR's commit message.